dotnet/src/Extensions/Planning.SequentialPlanner/SequentialPlanner.cs

```diff
@@ -62,11 +63,15 @@ public async Task<Plan> CreatePlanAsync(string goal, CancellationToken cancellat
 
         this._context.Variables.Update(goal);
 
-        var planResult = await this._functionFlowFunction.InvokeAsync(this._context, cancellationToken: cancellationToken).ConfigureAwait(false);
+        SKContext planResult;
 
-        if (planResult.ErrorOccurred)
+        try
         {
-            throw new SKException($"Error creating plan for goal: {planResult.LastException?.Message}", planResult.LastException);
+            planResult = await this._functionFlowFunction.InvokeAsync(this._context, cancellationToken: cancellationToken).ConfigureAwait(false);
+        }
+        catch (Exception ex)
+        {
+            throw new SKException($"Error creating plan for goal: {ex.Message}", ex);
```

Comment:
As discussed, we probably don't need to catch/throw here any longer.

Reply:
The try/catch is removed.